components/DownloadModal.js

```diff
@@ -3,12 +3,12 @@ export const DownloadModal = ({ setShowModal }) => {
     <>
       {/* This example requires Tailwind CSS v2.0+ */}
       <div
-        className="fixed z-10 inset-0 overflow-y-auto"
+        className="fixed inset-0 z-10 overflow-y-auto"
```

Comment:
I see you are using the same plugin as I do in VS Code ([Headwind](https://marketplace.visualstudio.com/items?itemName=heybourn.headwind)). I did not commit these changes when I submitted a PR (#15), because I was not sure if @katherinepeterson wants that.

Reply:
@natterstefan yeah I actually just forgot to have Headwind enabled when I did this project, so I told @nachoiacovino he could go in and format the entire project!